apps/coscrad-frontend-e2e/src/e2e/resource-index-to-detail-flows/notes-panel.e2e.cy.ts

```diff
@@ -0,0 +1,80 @@
+describe(`open self notes panel for resource`, () => {
+    const photographBaseRoute = `/Resources/Photographs/`;
+
+    const photographId = `9b1deb4d-3b7d-4bad-9bdd-2b0d7b110002`;
+
+    describe(`when opening the self notes panel for a resource`, () => {
+        beforeEach(() => {
+            cy.visit(`${photographBaseRoute}${photographId}`);
+        });
+
+        describe(`when the photograph detail full view has loaded`, () => {
+            it(`should expose the open note panel button`, () => {
+                cy.getByDataAttribute('TextSnippetIcon').should('exist');
+            });
+
+            it(`should not expose the panel`, () => {
+                cy.get('[data-testid="notes-panel"] > .MuiPaper-root').should('not.exist');
+            });
+
+            describe(`when the notes panel button is clicked`, () => {
+                beforeEach(() => {
+                    cy.getByDataAttribute('open-notes-panel-button').click();
+                });
+
+                it(`should open the notes panel`, () => {
+                    cy.get('[data-testid="notes-panel"]').should('be.visible');
+                });
+
+                describe(`when the notes panel is open`, () => {
+                    it(`should expose the notes panel close button`, () => {
+                        cy.getByDataAttribute('close-notes-panel-button').should('be.visible');
+                    });
+
+                    it(`should expose the notes panel expand button`, () => {
+                        cy.getByDataAttribute('expand-notes-panel-button').should('be.visible');
+                    });
+                });
+
+                /**
+                 * Note 1: this test may fail with different viewport heights, need to test further.
```

Comment:
I think these considerations are important for this particular test. We want to make sure that this complex experience is responsive.

Reply:
Yes!  At this stage the panels are not responsive.  Note panel basically works but the button display needs to be fixed.  The connected resources panel does not work.